### PR TITLE
Update usage of docker to oci-image resource type.

### PR DESCRIPTION
**Pull request by Veebers**

### CODE REVIEW

File: dependencies.tsv [hunk 1]
```diff
@@ -46,6 +46,7 @@ github.com/mattn/go-colorable	git	ed8eb9e318d7a84ce5915b495b7d35e0cfe7b5a8	2016-
 github.com/mattn/go-isatty	git	66b8e73f3f5cda9f96b69efd03dd3d7fc4a5cdb8	2016-08-06T12:27:52Z
 github.com/mattn/go-runewidth	git	d96d1bd051f2bd9e7e43d602782b37b93b1b5666	2015-11-18T07:21:59Z
 github.com/matttproud/golang_protobuf_extensions	git	c12348ce28de40eed0136aa2b644d0ee0650e56c	2016-04-24T11:30:07Z
+github.com/ncw/swift	git	b2a7479cf26fa841ff90dd932d0221cb5c50782d	2018-03-27T20:33:07Z
```
Comment (wallyworld) [inline dependencies.tsv:49]: DO we need this new dep?

**Review by wallyworld — APPROVED** [review]
lgtm apart from new dep

File: dependencies.tsv (hunk 1 above)
Comment (Veebers) [reply dependencies.tsv:49]: This is a transient dep from charmstore (which is a newish dep there too)

**Comment by Veebers** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-client-merge

**Pull request merged**